pkg/config/config_test_data.go

```diff
@@ -32,6 +32,7 @@ const (
     "worker-num": 3
   },
   "sink": {
+	"encoder-concurrency": 16,
```

Comment:
Is the indentation here consistent with the next line?

Reply:
yes, at the same level .